Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+{
+  "modmenu.dropInfo.line1": "FO modlarında §9✎§r veya §8✎§r, altlarda bulunan: §c*§r;",
+  "modmenu.dropInfo.line2": "diğerleri desteklenmemektedir.",
+  "modmenu.summaryTranslation.slight-gui-modifications": "Menüdeki animasyonlar ve modpaket sürümü\n§9✎§8 Hareketler, renkler, FPS göstergesi, ...",
+  "modmenu.summaryTranslation.advancementinfo": "Daha büyük ve akıllı başarım ekranı\n§9✎§8 Renkler, bilgi kutucuk büyüklüğü, ...",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Daha büyük başarım ekranı\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.animatica": "Optifine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bobby": "Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
+  "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
+  "modmenu.summaryTranslation.capes": "Optifine veya diğerlerinden pelerinler\n§9✎§8 Türler, tercih edilen tür, elitra, ...",
+  "modmenu.summaryTranslation.citresewn": "Optifine'ın özel eşyaları\n§9✎§8 Modu aç/kapa, bozuk paketlere izin ver, ...",
+  "modmenu.summaryTranslation.colormatic": "Optifine'ın özel renkleri\n§9✎§8 Gökyüzünü veya yer boşluğunu temizle, ışık yumuşatma",
+  "modmenu.summaryTranslation.continuity": "Optifine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.cullleaves": "Optifine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
+  "modmenu.summaryTranslation.cem": "Optifine'ın özel varlıkları (alfa)\n§9✎§8 Optifine desteği, model düzeltimi",
+  "modmenu.summaryTranslation.dcch": "Farklı dünyalardan yazım geçmişini tut\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.dynamicfps": "Arkaplandayken FPS'i düşür\n§9✎§8 Modu aç/kapa, üzerindeyken normale çevir, ...",
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandıklar, tabelalar, yataklar vb.\n§9✎§8 Geliştirilecek bloklar",
```

Comment:
Context: the mod makes chests, signs, beds, bells, shulker boxes render faster. It is possible to toggle each optimization off in options, and choose the method with which the blocks will be optimized.